pilot/pkg/networking/core/cluster_builder.go

```diff
@@ -374,6 +377,11 @@ func (cb *ClusterBuilder) buildInboundCluster(clusterPort int, bind string,
 			string(instance.Service.Hostname), "", instance.Port.ServicePort, clusterPort, &instance.Service.Attributes)
 	}
 
+	if clusterType == cluster.Cluster_ORIGINAL_DST {
+		// Disable cleanup for inbound clusters - set to Max possible duration.
+		localCluster.cluster.CleanupInterval = durationpb.New(time.Duration(maxSecondsValue) * time.Second)
```

Comment:
is ·durationpb.New·  a special value, not quite underestand.

durationpb.New(1 year)?

Reply:
Any higher number is OK